### PR TITLE
[Ruby 2.5] Top-level constant look-up is removed

### PR DESCRIPTION
**Pull request by andrykonchin**
https://bugs.ruby-lang.org/issues/11547

Added specs are very short and I have no idea what other cases we can add

### CODE REVIEW

File: language/constants_spec.rb [hunk 1]
```diff
@@ -425,6 +425,26 @@ module ConstantSpecs
...
+  end
+
+  it "searches Object unsuccessfully when starts from module" do
+    ->() { Enumerable::Hash }.should raise_error(NameError)
+  end
+
+  ruby_version_is "2.5" do
+    it "does not search Object after searching other scopes" do
+      ->() { String::Hash }.should raise_error(NameError)
+    end
```
Comment (eregon) [inline language/constants_spec.rb]: Could you move this one below the first spec so the version guards are next to each other?

Comment (andrykonchin) [reply language/constants_spec.rb]: @eregon Done

Comment (eregon) [reply language/constants_spec.rb]: A "when starts from class" is missing here. One nice way would be to move these 2 specs in a nested "on a class" describe block.

Comment (andrykonchin) [reply language/constants_spec.rb]: @eregon done

**Comment by eregon** [comment]
Thanks! I think that's all there is to test here :)

**Pull request merged**